content/en/docs/concepts/architecture/nodes.md

```diff
@@ -115,9 +115,9 @@ are enabled, kubelets are only authorized to create/modify their own Node resour
 {{< note >}}
 As mentioned in the [Node name uniqueness](#node-name-uniqueness) section,
 when Node configuration needs to be updated, it is a good practice to re-register
-the node with the API server. For example, if the kubelet being restarted with
-the new set of `--node-labels`, but the same Node name is used, the change will
-not take an effect, as labels are being set on the Node registration.
+the node with the API server. For example, if the kubelet is being restarted with
+a new set of `--node-labels`, but the same Node name is used, the change will
+not take effect, as labels are only set (or reset) upon Node registration with the API server.
```

Comment:
What does `reset` mean here?

Reply:
@kurtado please, address this review comment.

Reply:
I intended to convey that the labels are "set" upon initial registration, and "reset" upon subsequent registrations with the API server, but I can remove the `(or reset)` if that might confuse readers.

Reply:
Try “modified” in place of “reset”?

Reply:
Will do, thanks!